src/controllers/main/main.ts

```diff
@@ -493,8 +491,13 @@ export class MainController extends EventEmitter {
     }
 
     await this.resolveUserRequest({ hash: signedMessage.signature }, signedMessage.fromActionId)
-
-    this.onSignSuccess(signedMessage.content.kind === 'typedMessage' ? 'typed-data' : 'message')
+    await this.#notificationManager.create({
+      title: 'Successfully signed',
+      message:
+        signedMessage.content.kind === 'typedMessage'
+          ? 'TypedData was successfully signed'
+          : 'Message was successfully signed'
+    })
```

Comment:
We can take the chance to update the text (since the title and message repeat themselves). How about simply:

title: Done!
message: The message was signed.